ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -22,21 +22,38 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     Ok(views.html.app("Recipeasy"))
   }
 
-  def curateRecipePage = Action { implicit request =>
+  def curateNewRecipe = Action { implicit request =>
     val newRecipe = db.getNewRecipe
     newRecipe match {
+      case Some(r) => Redirect(routes.Application.curateRecipe(r.id))
+      case None => NotFound
+    }
+  }
+
+  def curateRecipe(id: String) = Action { implicit request =>
+    val recipe = db.getRecipe(id)
+    view(recipe, editable = true)
+  }
+
+  def viewRecipe(id: String) = Action { implicit request =>
+    val recipe = db.getRecipe(id)
```

Comment:
Need to write this method: https://github.com/guardian/recipeasy/blob/master/common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala#L181 and add a method to extract from `curated_recipe` table.